tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -134,6 +148,11 @@ trait VirtualScalaJSIRFile extends VirtualFile {
   def infoAndTree: (ir.Infos.ClassInfo, ir.Trees.ClassDef)
 }
 
+trait VirtualRelativeIRFile extends VirtualScalaJSIRFile
+    with RelativeVirtualFile with IRContainer {
```

Comment:
First time I wish for an extension-type system where all `VirtualScalaJSIRFile with RelativeVirtualFile` automatically become `<: IRContainer` :-/